### PR TITLE
feat: Add contact details in services

### PR DESCRIPTION
**Pull request by rut1303**

### CODE REVIEW

File: src/Pages/Services/BookDistribution/constant.ts [hunk 1]
```diff
@@ -49,4 +49,14 @@ export const bookDistributionImages = [
 
 export const bookDistribution = [
   ' "Books are the foundation", as Srila Prabhupad always said. The significance of disseminating Srila Prabhupad’s transcendental literature cannot be overstated. On numerous occasions, it has been observed that Srila Prabhupad prioritised the publication and distribution of books over the construction of temples or any other societal activity. The mission of Sri Chaitanya Mahaprabhu was spread throughout the world thanks to Srila Prabhupad’s transcendental books. This was also the state of mind of his spiritual master, Srila Bhakti Siddhanta Saraswati Thakur Prabhupad, who wrote volumes upon volumes of transcendental literature. As a result, book distribution remains one of the most important services provided by devotees.',
+
+  "Contact Details:"
+];
+
+export const contactDetailes1=[
```
Comment (vaidehi27) [inline src/Pages/Services/BookDistribution/constant.ts]:
```suggestion
export const contactDetails1=[
```

File: src/Pages/Services/PageLayout/index.tsx [hunk 2]
```diff
@@ -7,17 +8,22 @@ import {
...
 import titlebBaseImage from "../../../assets/images/title-base.png";
 import { PageCarousel } from "./PageCarousel";
+import React from "react";
 
 interface ServiceDescriptionProps {
   title: string;
   content: string[];
   imageUrl: { id: number; image: string }[];
+  contactNumber: { id: number; number: string }[];
+  emailId: string[];
```
Comment (vaidehi27) [inline src/Pages/Services/PageLayout/index.tsx]:
```suggestion
  emailId?: string[];
```

Make it optional and then you will not need to pass emailId if not required

File: src/Pages/Services/BookDistribution/constant.ts (hunk 1 above)
Comment (rut1303) [reply src/Pages/Services/BookDistribution/constant.ts]: Resolve

File: src/Pages/Services/BookDistribution/index.tsx [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
+} from "./constant";
 
 export const BookDistribution: React.FC = () => {
   return (
     <ServiceDescription
       title="Book Distribution"
       content={bookDistribution}
       imageUrl={bookDistributionImages}
+      contactNumber={contactDetails}
+      emailId={[]}
```
Comment (vaidehi27) [inline src/Pages/Services/BookDistribution/index.tsx]: you don't need to pass empty emailId because it's optional. Update it everywhere.

Comment (rut1303) [reply src/Pages/Services/BookDistribution/index.tsx]: Resolved it.

File: src/Pages/Services/PageLayout/style.ts [hunk 4]
```diff
@@ -129,3 +129,12 @@ export const PageReviews = styled.div`
...
   }
 `;
+
+export const ContactLink = styled.a`
+color: #ad0d0d;
+text-decoration: none;
+
+&:hover{
+  color: #000;
+}
```
Comment (vaidehi27) [inline src/Pages/Services/PageLayout/style.ts]: The format is not applied. @Priyansh1825 I thought we have checked in place. How was Rut able to commit without formatting?

Comment (Priyansh1825) [reply src/Pages/Services/PageLayout/style.ts]: Done

File: src/Pages/Services/IskconYouthForum/constant.ts [hunk 5]
```diff
@@ -44,8 +44,17 @@ export const iskconYouthForum = [
...
+    id: 1,
+    number: "+91 99987 14977",
+  },
+  {
+    id: 2,
+    number: "+91 74051 60363",
+  },
 ];
+
+export const emailDetails = ["Email: abhishekmishra2@gmail.com"];
```
Comment (vaidehi27) [inline src/Pages/Services/IskconYouthForum/constant.ts]:
```suggestion
export const emailDetails = ["abhishekmishra2@gmail.com"];
```

**Review by vaidehi27 — APPROVED** [review]

File: src/Pages/Services/IskconYouthForum/constant.ts (hunk 5 above)
Comment (Priyansh1825) [reply src/Pages/Services/IskconYouthForum/constant.ts]: Done

**Pull request merged**